tests/regtest.py

```diff
@@ -486,6 +486,9 @@ def checkCapsSupport(self):
 	def getChkstatPath(self):
 		return "/usr/local/bin/chkstat"
 
+	def getChkstatConfigRoot(self):
+		return "/usr/local"
+
```

Comment:
I deliberately made them methods to allow potentially more complex logic in the future. Judging from the evolution this whole test setup made by now it's quite conceivable that additional complexity happens ;)

Reply:
I'm usually not a fan of this kind of proactive complexity, because way too often later developments are not perfectly predicted so you just end up with an even more complex result in the end. (The buzzword for this is YAGNI.)

This is just a simple function here though, so there's not much point in bikeshedding this too much.